### PR TITLE
Ag 1939 duplicate email validation

### PR DESCRIPTION
**Pull request by agillSoliel**
## Description

Fixed issue 1939 Duplicate email error. Current development environment does not throw an error when the user puts the same email for their work and home (page: /form/identification/contacts). I have added error handling for this issue so the user is prompted to give two different emails if they enter a duplicate.

## Checklist for Reviewer

- [ ] Review code changes
- [ ] Review changes for Database effects
- [ ] Verify change works in IE browser

More details about this can be found in [docs/review.md](docs/review.md)

### CODE REVIEW

File: src/components/Section/Identification/ContactInformation/ContactInformation.jsx [hunk 1]
```diff
@@ -15,6 +15,7 @@ import { IDENTIFICATION, IDENTIFICATION_CONTACTS } from 'config/formSections/ide
 
 import connectSubsection from 'components/Section/shared/SubsectionConnector'
 import Subsection from 'components/Section/shared/Subsection'
+import Generic from '../../../Form/Generic'
```
Comment (suzubara) [inline src/components/Section/Identification/ContactInformation/ContactInformation.jsx]: It doesn't look like this import is used

File: src/config/locales/en/error.js [hunk 2]
```diff
@@ -1137,6 +1137,10 @@ export const error = {
       title: 'Your response is required',
       message: '',
     },
+    duplicate: {
+      title: 'There is a problem with your emails',
+      message: 'Please provide different emails.'
```
Comment (suzubara) [inline src/config/locales/en/error.js]: @michaelccata flagging this as new error message copy, does this need to be approved?

Comment (michaelccata) [reply src/config/locales/en/error.js]: Thanks for flagging @suzubara 
LGTM

File: src/components/Section/Identification/ContactInformation/ContactInformation.jsx (hunk 1 above)
Comment (agillSoliel) [reply src/components/Section/Identification/ContactInformation/ContactInformation.jsx]: Pushed a fix to remove this from the code.

**Comment by christopherhuii** [comment]
We gotta get some additional tests in before merging. That's a steep drop-off in test coverage.

**Comment by ryanhofdotgov** [comment]
I've exported this branch as a patch and attached it to EN-1939 with some recommended next steps.

**Pull request closed without merging**